ext/DocumenterVitepressDocumenterCitationsExt.jl

```diff
@@ -0,0 +1,100 @@
+module DocumenterVitepressDocumenterCitationsExt
+
+using DocumenterCitations, DocumenterVitepress
+
+import DocumenterVitepress as DV
+using Documenter: MarkdownAST
+
+# TODO: 
+# - List style (rendered vs unrendered)
+# - Loose vs tight lists
+# - handle :dl properly, we cannot use Markdown for this, since it cannot be represented
+#   using pure Markdown. We need to insert HTML directly.
+#   At the moment, we treat :dl as :ol
```

Comment:
We should be able to do that, I think Vitepress accepts pure HTML.  I'm just not sure if the Markdown will still be picked up.  We'll have to test but that can be done later.

Reply:
Sure